tests/test_api.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+import pyblish.api
+
+
+def test_environment_host_registration():
```

Comment:
Thanks for this.

I'm not sure this is the right approach. Maybe a better test would be to include a plug-in that you've associated to a host, and test for whether it runs as expected? That way you could test it both via `util.publish` and the CLI equivalent.